CODE/style.css

```diff
@@ -0,0 +1,130 @@
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #d2d3d3;
+
+}
+
+/* pull QUote*/
+blockquote {
+    text-align: left;
+    margin: 0;
+}
+
+
+header {
+    background-image: url(images/maxresdefault.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+/*Title and Author*/
+h1 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 60px;
+    text-align: center;
+    color: #ce5454f8;
+}
+
+h2 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 45px;
+    text-align: center;
+    color: #ce5454f8;
+}
+
+h3 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 30px;
+    text-align: left;
+}
+
+/*article text*/
+p1 {
```

Comment:
Just like in the HTML, you need to correct this since `p1` is not a valid HTML element.